libsnowflakeclient/deps-build/darwin/aws/include/aws/auth/auth.h

```diff
@@ -0,0 +1,91 @@
+#ifndef AWS_AUTH_AUTH_H
```

Comment:
Do we need header files of the libsnowflakeclient dependencies? can we drop them?

Reply:
done